src/containerapp/azext_containerapp/_validators.py

```diff
@@ -215,3 +220,49 @@ def validate_timeout_in_seconds(cmd, namespace):
     if timeout_in_seconds is not None:
         if timeout_in_seconds < 0 or timeout_in_seconds > 60:
             raise ValidationError("timeout in seconds must be in range [0, 60].")
+
+
+def validate_debug(cmd, namespace):
+    print("Validating...")
+    revision_already_set = bool(namespace.revision)
+    replica_already_set = bool(namespace.replica)
+    container_already_set = bool(namespace.container)
+    _set_debug_defaults(cmd, namespace)
+    if revision_already_set:
+        _validate_revision_exists(cmd, namespace)
+    if replica_already_set:
+        _validate_replica_exists(cmd, namespace)
+    if container_already_set:
+        _validate_container_exists(cmd, namespace)
+
+
+def _set_debug_defaults(cmd, namespace):
+    app = ContainerAppPreviewClient.show(cmd, namespace.resource_group_name, namespace.name)

```

Comment:
we can list revisions here and see: 1) if we got 404 error or None object then the container app doesnt exist 2) if we got an empty array then the container app exists but doesnt have any revisions
and the revision list data could be reused in checking user input parameter "targetContainer"

Reply:
IMO. List revisions will return all revision, this is not a good choice. If customer has lots of revisions, the payload will be big enough.

Reply:
normally the number of revisions in a single container app wont be too much. currently only 3.7% of container apps are in multiple active revision mode.
btw if you take a closer look at the container app payload and revision payload you will see container app is much larger than just a revision.

Reply:
It seems this API will return all the revisions, which are active or not active.